api/ticket.rst

```diff
@@ -1,17 +1,18 @@
 Ticket endpoints
```

Comment:
We should also change this to "Ticket Endpoints" for proper title casing (see comment below).